v2/gcs-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/dao/BaseDao.java

```diff
@@ -88,13 +72,14 @@ public void batchWrite(List<String> batchStatements) throws SQLException {
         }
         statement.executeBatch();
         status = true;
+      } catch (org.postgresql.util.PSQLException e) {
```

Comment:
Since Postgres JDBC driver does not throw specific exceptions, but the error type is available in the SQL State parameter of the error. Accordingly I have updated the logic to filter based on the transient connection error codes.

Refer: 

https://www.postgresql.org/docs/current/errcodes-appendix.html